tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -868,16 +873,17 @@ async def test_nft_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
     )
     assert res["success"]
 
-    for _ in range(3):
-        await farm_transaction_block(full_node_api, wallet_1_node)
+    spend_bundle = SpendBundle.from_json_dict(json_dict=res["spend_bundle"])
+
+    await farm_transaction(full_node_api, wallet_1_node, spend_bundle)
```

Comment:
why did you change from farm_transaction_block to farm_transaction ?
if it is more consistent why not the other tests as well ?

Reply:
I was just hacking around at broken stuff, not trying to correct everything.  I think the intent here is to process the transaction and then check some stuff after that.  Instead of guessing the minimum number of blocks required to always make that happen on all systems, it seemed more robust to use the function that would do we what want.  Get the transaction processed.  (though 'farm' is an odd word for that)

All of this not-really-coverage stuff should probably be taken as 'inspiration' for a separate pr to make improvements in these and similar areas instead of just being a side note here.

Reply:
i would leave it then and lets do a pr where we overhaul all these flakey tests instead